packages/network/src/logic/RtcSignaller.ts

```diff
@@ -97,17 +102,24 @@ export class RtcSignaller {
         })
     }
 
-    onLocalDescription(routerId: string, targetPeerId: string, type: DescriptionType, description: string): void {
-        this.trackerNode.sendLocalDescription(routerId, targetPeerId, this.peerInfo, type, description)
+    sendRtcOffer(trackerId: string, targetPeerId: string, connectionId: string, description: string): void {
+        this.trackerNode.sendRtcOffer(trackerId, targetPeerId, connectionId, this.peerInfo, description)
+            .catch((err: Error) => {
+                this.logger.debug('failed to sendRtcOffer via %s due to %s', trackerId, err) // TODO: better?
+            })
+    }
+
+    sendRtcAnswer(trackerId: string, targetPeerId: string, connectionId: string, description: string): void {
```

Comment:
`trackerId` => `routerId` for naming to be consistent with rest of methods in this class. (Alternatively re-name occurrences of `routerId` => `trackerId`)

Reply:
renamed to routerId